src/tests/home.test.js

```diff
@@ -0,0 +1,9 @@
+import renderer from 'react-test-renderer';
+import React from 'react';
+import Home from '../components/Home';
+
+test('Home component snapshot', () => {
+  const component = renderer.create(<Home />);
+  const tree = component.toJSON();
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
- [ ] As the project requirement you should create unit tests for all the react components. So please create tests for all other components as well. You did not test all the component that is in the component folder.